specs/IsPinchZoomEnabled.md

```diff
@@ -0,0 +1,71 @@
+# Background
+
+There are two types of zoom in Chromium: Browser Zoom and Pinch-Zoom:
```

Comment:
I understood this better when you described it to me in the ADO PR. For both kinds of zoom please include its behavior (the difference is in what happens to the viewport or something right?), the APIs applicable for the zoom, and then how the end user can change it.